cwl_utils/parser/__init__.py

```diff
@@ -5,6 +5,7 @@
 from . import cwl_v1_0 as cwl_v1_0
 from . import cwl_v1_1 as cwl_v1_1
 from . import cwl_v1_2 as cwl_v1_2
+from . import latest as latest
```

Comment:
```suggestion
```

Unless you are going to add code as part of this PR that uses `latest` in `cwl_utils/parser/__init__.py`